---
title: GSMR
author: Gibran Hemani
date: '2018-01-15'
comments:
  utterances:
    repo: open-mr-project/siteComments
data:
  id: 11.0
  start_time: 1.706184e+09
  completion_time: 1.7061846e+09
  email: anonymous
  name: .na
  last_modified_time: .na
  primary_entry_contact: Gibran Hemani
  primary_entry_contact_email_address: g.hemani@bristol.ac.uk
  primary_method_contact: Jian Yang
  primary_method_contact_email_address: jian.yang@westlake.edu.cn
  method_name: Generalised summary-data-based mendelian randomisation
  abbreviated_method_name: GSMR
  publication_doi: https://doi.org/10.1038/s41467-017-02317-2
  code_repository_url: https://yanglab.westlake.edu.cn/software/gsmr/
  code_language: R
  method_website: https://yanglab.westlake.edu.cn/software/gsmr/
  other_url: none
  which_analysis_types_is_the_method_used_for: Univariable MR (one exposure, one outcome);Bi-directional
    MR;Multivariable MR (multiple exposures, one outcome);
  specific_scientific_context_if_any: Complex traits;Molecular traits;
  what_types_of_exposure_variables_can_be_used_if_network: Quantitative/continuous;Binary;
  what_types_of_outcome_variables_can_be_used_if_network: Quantitative/continuous;Binary;
  what_data_types_can_the_method_be_used_with: GWAS summary statistics for both exposure(s)
    and outcome(s) (or all traits if Network MR);
  which_mr_conditions__assumptions_are_requiredrelaxed_by_the_method: The method accounts
    for balanced (uncorrelated) pleiotropy or pleiotropy that meets the Instrument
    strength independent of direct effect (InSIDE) assumption. ;
  brief_description: GSMR performs a multi-SNP Mendelian randomization analysis using
    summary-level data from genome-wide association studies, allowing for outlier
    removal to account for pleiotropy, multivariable models, correlation of instruments
    and bi-directional effect estimation.
  title: GSMR
  author: Gibran Hemani
  slug: gsmr
  pubdate: '2018-01-15'
  uvmr_class: greenbg
  mvmr_class: greenbg
  bidirectional_class: greenbg
  nonlinear_class: greybg
  network_class: greybg
  ii_class: greybg
  ss_class: greenbg
  fam_class: greybg
  is_class: greybg
  si_class: greybg
  exquant_class: greenbg
  exbin_class: greenbg
  extte_class: greybg
  outquant_class: greenbg
  outbin_class: greenbg
  outtte_class: greybg
  weak_inst_symbol: '&cross;'
  winners_curse_symbol: '&cross;'
  sample_overlap_symbol: '&cross;'
  confounding_symbol: '&cross;'
  assortative_mating_symbol: '&cross;'
  uhp_symbol: '&check;'
  chp_symbol: '&cross;'
  colliding_symbol: '&cross;'
  anc_diff_symbol: '&cross;'
  weak_inst_class: crossclass
  winners_curse_class: crossclass
  sample_overlap_class: crossclass
  confounding_class: crossclass
  assortative_mating_class: crossclass
  uhp_class: checkclass
  chp_class: crossclass
  colliding_class: crossclass
  anc_diff_class: crossclass
---

<style>

.greenbg {
    color: white;
    background-color: #00796B;
}

.greybg {
    color: white;
    background-color: #808080;
}

.checkclass {
    color: #00796B;
    font-weight: 800;
    font-size: 2.5em;
}

.crossclass {
    color: #808080;
    font-weight: 600;
    font-size: 2.5em;
}



</style>
## Resources

[Code]({{< meta data.code_repository_url >}}) (Language: {{< meta data.code_language >}}) <span style="margin-right: 30px;"></span> [Website]({{< meta data.method_website >}}) <span style="margin-right: 30px;"></span> [Paper]({{< meta data.publication_doi >}}) (Pub Date: {{< meta data.pubdate >}})

Other URLS: {{< meta data.other_url >}}

Method Contact: {{< meta data.primary_method_contact >}} ( {{< meta data.primary_method_contact_email_address >}})

Entry Contact: {{< meta data.primary_entry_contact >}} ( {{< meta data.primary_entry_contact_email_address >}})

## Context

```{r, echo=FALSE, results='asis'}
```

Analysis Type:  <span class = "`r rmarkdown::metadata$data$uvmr_class`">UVMR</span> <span style="margin-right: 10px;"></span> <span class = "`r rmarkdown::metadata$data$mvmr_class`">MVMR</span> <span style="margin-right: 10px;"></span> <span class="`r rmarkdown::metadata$data$network_class`">Network MR</span>  <span style="margin-right: 10px;"></span> <span class = "`r rmarkdown::metadata$data$bidirectional_class`">Bi-Directional MR</span></span><span style="margin-right: 10px;"></span> <span class="`r rmarkdown::metadata$data$nonlinear_class`">Non-Linear</span>


Input Data Types:  <span class = "`r rmarkdown::metadata$data$ii_class`">Ind/Ind</span> <span style="margin-right: 10px;"></span> <span class = "`r rmarkdown::metadata$data$is_class`">Ind/SS</span> <span style="margin-right: 10px;"></span> <span class="`r rmarkdown::metadata$data$si_class`">SS/Ind</span>  <span style="margin-right: 10px;"></span> <span class = "`r rmarkdown::metadata$data$ss_class`">SS/SS</span><span style="margin-right: 10px;"></span><span class = "`r rmarkdown::metadata$data$fam_class`">family</span>

Exposure Trait Types:  <span class="`r rmarkdown::metadata$data$exquant_class`">Quantitative</span> <span style="margin-right: 10px;"></span> <span class="`r rmarkdown::metadata$data$exbin_class`">Binary</span> <span style="margin-right: 10px;"></span> <span class="`r rmarkdown::metadata$data$extte_class`">Time to event</span> 

Outcome Trait Types: <span class="`r rmarkdown::metadata$data$outquant_class`">Quantitative</span> <span style="margin-right: 10px;"></span> <span class="`r rmarkdown::metadata$data$outbin_class`">Binary</span> <span style="margin-right: 10px;"></span> <span class="`r rmarkdown::metadata$data$outtte_class`">Time to event</span> 

## Assumptions and Sources of Bias

| Source of Bias | Addressed |
|:-----------|:--------:|
| Weak instruments | <span class = "`r rmarkdown::metadata$data$weak_inst_class`">{{< meta data.weak_inst_symbol >}}</span> | 
| Winner's curse | <span class = "`r rmarkdown::metadata$data$winners_curse_class`">{{< meta data.winners_curse_symbol >}}</span>  |  
| Sample overlap | <span class = "`r rmarkdown::metadata$data$sample_overlap_class`">{{< meta data.sample_overlap_symbol >}}</span> |
| Uncorrelated horizontal pleiotropy | <span class = "`r rmarkdown::metadata$data$uhp_class`">{{< meta data.uhp_symbol >}}</span> |
| Correlated horizontal pleiotropy |  <span class = "`r rmarkdown::metadata$data$chp_class`">{{< meta data.chp_symbol >}}</span> | 
| Ancestry differences in samples | <span class = "`r rmarkdown::metadata$data$anc_diff_class`">{{< meta data.anc_diff_symbol >}}</span>| 
| Residual confounding in GWAS | <span class = "`r rmarkdown::metadata$data$confounding_class`">{{< meta data.confounding_symbol >}}</span>| 
| Cross-trait assortative mating |<span class = "`r rmarkdown::metadata$data$assortative_mating_class`">{{< meta data.assortative_mating_symbol >}}</span>| 
| Index-event/conditioning on heritable trait | <span class = "`r rmarkdown::metadata$data$colliding_class`">{{< meta data.colliding_symbol >}}</span>| 


## Description

{{< meta data.brief_description >}}

